doc/BENCHMARKING.md

```diff
@@ -0,0 +1,45 @@
+
+In general, we run each IO operation for 30 seconds against a 100 GiB file. But there are some variants in configuration we also want to test to see how Mountpoint would perform with these configurations. Here is the list of all the variants we have tested.
+
+* **four_threads**: running the workload concurrently by spawning four threads to do the same job.
+* **direct_io**: bypassing linux page cache by opening the files with `O_DIRECT` option. This option is only available on Linux.
+* **small_file**: run the IO operation against smaller files (5 MiB instead of 100 GiB).
+
+### Regression Testing
+Our CI runs the benchmark automatically for any new commits to the main branch or specific pull requests that we have reviewed and tagged with **performance** label. Every benchmark from the CI workflow will be running on `m5n.24xlarge` EC2 instances (100 Gbps network speed) with Ubuntu 22.04 in us-east-1 against a bucket in us-east-1.
+
+We keep the records of benchmarking results in `gh-pages` branch and it is available for viewing [here](https://awslabs.github.io/mountpoint-s3/dev/bench/).
+
+### Running the benchmark
+Our benchmark script is designed for CI testing only, but in case you wish to run it manually, you should be able to do so by following these steps.
+
+1. Install dependencies by running below commands for Ubuntu.
+
+        sudo apt-get update
+        sudo apt-get -y install cmake libclang-dev libunwind-dev pkg-config jq fio
+        sudo apt-get -y install fuse libfuse-dev
+        echo 'user_allow_other' | sudo tee -a /etc/fuse.conf
+
+2. Create the bench files manually in your bucket. The size of the files must be exactly the same as the size defined in fio configuration files. The easiest way to do this is running fio against your local file system first to let fio create the files for you, and then upload them to your S3 bucket using the AWS CLI. For example:
+
+        fio --directory=your_local_dir --filename=your_file_name mountpoint-s3/scripts/fio/seq_read_small.fio
+        aws s3 cp your_local_dir/your_file_name s3://your_s3_bucket/prefix_path/
+
+3. Set environment variables related to the benchmark. There are four required environment variables you need to set in order to run the benchmark.
+
+        export S3_BUCKET_NAME=bucket_name
+        export S3_BUCKET_TEST_PREFIX=prefix_path/
+        export S3_BUCKET_BENCH_FILE=bench_file_name
+        export S3_BUCKET_SMALL_BENCH_FILE=small_bench_file_name
+
+4. Run the [benchmark script](https://github.com/awslabs/mountpoint-s3/blob/main/mountpoint-s3/scripts/fs_bench.sh).
```

Comment:
Prefer relative link here

```suggestion
4. Run the [benchmark script](../mountpoint-s3/scripts/fs_bench.sh).
```

Reply:
Changed to relative link, note that we won't be able to view it in the PR because the branch doesn't exist in upstream repo.